src/site/includes/header.html

```diff
@@ -59,6 +59,10 @@
 <link rel='stylesheet' href='/assets/css/ie.css'>
 <![endif]-->
 
+  {% if botframework_cdn %}
+    <script src="{{ botframework_cdn }}"></script>
+  {% endif %}
```

Comment:
I used the name `botframework_cdn` because I wanted to portray that it has a specific use case and shouldn't be reused